packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -532,6 +532,31 @@ export interface IContainerRuntimeOptions {
 	readonly explicitSchemaControl?: boolean;
 }
 
+/**
+ * Internal extension of @see IContainerRuntimeOptions
+ *
+ * These options are not available to consumers when creating a new container runtime,
+ * but we do need to expose them for internal use, e.g. when configuring the container runtime
+ * to ensure compability with older versions.
+ *
+ * @internal
```

Comment:
why internal? what other package will use this? can it just be exported with the package for unit tests only?

Reply:
It's for `compatibilityConfiguration.ts`, for 1.x compat